playground/frontend/pubspec.yaml

```diff
@@ -44,6 +44,7 @@ environment:
 # the latest version available on pub.dev. To see which dependencies have newer
 # versions available, run `flutter pub outdated`.
 dependencies:
+  akvelon_flutter_issue_106664_workaround: ^0.1.1
```

Comment:
Instead of publishing a package on pub.dev, could you create a local one and reference?  One could also reference a depedency via local or GitHub references: https://dart.dev/tools/pub/dependencies.  If a pub package is absolutely needed, I noticed two issues flagged in the scores:  https://pub.dev/packages/akvelon_flutter_issue_106664_workaround/score

Reply:
I have updated the package. It now have the max score, and the source is public here:
https://github.com/akvelon/flutter-issue-106664-workaround